app/workers/deletes.php

```diff
@@ -142,6 +221,49 @@ protected function deleteById(Document $document, Database $database, callable $
         Authorization::reset();
     }
 
+    protected function deleteForProjectIds(callable $callback)
+    {
+        $count = 0;
+        $chunk = 0;
+        $limit = 50;
+        $projects = [];
+        $sum = $limit;
+
+        $executionStart = \microtime(true);
+        
+        while($sum === $limit) {
+            $chunk++;
+
+            Authorization::disable();
+            $projects = $this->getConsoleDB()->getCollection([
+                'limit' => $limit,
+                'offset' => $count,
+                'orderField' => '$id',
```

Comment:
Why do we order here? 🤔

Reply:
@christyjacob4, @TorstenDittmann is right, no need for this anymore. Actually, the default order  (when passing no order setting) has a slightly better performance.

Reply:
Okay understood. Thanks 👍